### PR TITLE
Add LZ PhysiX PCB

### PR DESCRIPTION
**Pull request by hineybush**
## Description

Add PCB for LZ PhysiX 

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/hineybush/physix/keymaps/default/config.h [hunk 1]
```diff
@@ -0,0 +1,19 @@
+/* Copyright 2019 hineybush
```
Comment (fauxpark) [inline keyboards/hineybush/physix/keymaps/default/config.h]: This file can be deleted.

File: keyboards/hineybush/physix/keymaps/default/keymap.c [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+}
+
+void matrix_scan_user(void) {
+
+}
+
+bool led_update_user(led_t led_state) {
+    return true;
+}
+*/
```
Comment (fauxpark) [inline keyboards/hineybush/physix/keymaps/default/keymap.c]: These functions can be removed.

File: keyboards/hineybush/physix/physix.c [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+    matrix_scan_user();
+}
+
+bool process_record_kb(uint16_t keycode, keyrecord_t *record) {
+    // put your per-action keyboard code here
+    // runs for every action, just before processing by the firmware
+
+    return process_record_user(keycode, record);
+}
```
Comment (fauxpark) [inline keyboards/hineybush/physix/physix.c]: These functions can be removed.

File: keyboards/hineybush/physix/readme.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
+# physix
+
+![PhysiX](https://geekhack.org/index.php?topic=103049.0)
```
Comment (fauxpark) [inline keyboards/hineybush/physix/readme.md]: This needs to be an actual image.

File: keyboards/hineybush/physix/rules.mk [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+# If you don't know the bootloader type, then you can specify the
+# Boot Section Size in *bytes* by uncommenting out the OPT_DEFS line
+# Otherwise, delete this section
+#   Teensy halfKay      512
+#   Teensy++ halfKay    1024
+#   Atmel DFU loader    4096
+#   LUFA bootloader     4096
+#   USBaspLoader        2048
+# OPT_DEFS += -DBOOTLOADER_SIZE=4096
+
```
Comment (fauxpark) [inline keyboards/hineybush/physix/rules.mk]: Please delete this block.

File: keyboards/hineybush/physix/readme.md (hunk 4 above)
Comment (hineybush) [reply keyboards/hineybush/physix/readme.md]: PCBs and cases in production, will update once they're available

File: keyboards/hineybush/physix/info.json [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+        }
+    }
+}
+
+
+
+
+
+
```
Comment (fauxpark) [inline keyboards/hineybush/physix/info.json]: This won't break anything, but there's a few too many newlines here ;)

**Pull request merged**